orchestrator/config/processes_manager.go

```diff
@@ -11,15 +11,21 @@ type ProcessArg struct {
 	Value string `yaml:"value"` //Optional value for the arg. If not provided, it fallsback to gloabl params
 }
 
+// ProcessesArgs Represents set of arguments passed to processes
+type ProcessesArgs struct {
+	GroupArgs      []ProcessArg   `yaml:"group"`   //Arguments passed to all processes
+	IndividualArgs [][]ProcessArg `yaml:"process"` //Arguments passed to specific process
```

Comment:
Can we remove the `Args` part? I mean it's already `ProcessGroupConfig.Args`. So it would be `ProcessGroupConfig.Args.Group` instead of `ProcessGroupConfig.Args.GroupArgs`. I would also rename `Individual` to something else, maybe `Process`.